python/lsst/afw/table/_syntax.py

```diff
@@ -202,3 +202,70 @@ def processArray(a):
         else:
             d[name] = processArray(self.get(schemaItem.key))
     return d
+
+def BaseCatalog_asAstropy(self, cls=None, copy=False, unviewable="copy"):
```

Comment:
I think my worry about confusion comes from the fact that I [write `NumPy` in CamelCase code](https://github.com/lsst/ndarray/blob/master/python/ndarray.i#L41) even though it's `import numpy`.

But the NumPy and Astropy documentation and web pages seem to be quite internally consistent on the capitalization of the "p" while being unfortunately inconsistent with each other, so I maintain it's not my fault.